imagedephi/gui.py

```diff
@@ -27,40 +44,25 @@ def select_directory(
     input_directory: Path = Path("/"),  # noqa: B008
     output_directory: Path = Path("/"),  # noqa: B008
 ):
+    Path("")
```

Comment:
@brianhelba This shouldn't be committed, remove it.

Reply:
removed in d6e4e2812808229709879d48721e30756d7fc1bd